### ISSUE TITLE
Failed to activate the execute-as-ruby package

### ISSUE THREAD

**Issue by DomT4** [torsten/execute-as-ruby#5]
1. Open Atom. (Error instantly pops up)

**Atom Version**: 0.194.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.10.4
**Thrown From**: [execute-as-ruby](https://github.com/torsten/execute-as-ruby) package, v0.1.1

### Stack Trace

Failed to activate the execute-as-ruby package

```
At Cannot read property 'command' of undefined

TypeError: Cannot read property 'command' of undefined
    at Object.module.exports.activate (/Users/Dominyk/.atom/packages/execute-as-ruby/lib/execute-as-ruby.coffee:8:32)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:222:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:203:30
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:195:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:434:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:418:29
    at Config.module.exports.Config.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:311:16)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:413:19)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:394:46)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "one-dark-syntax"
    ],
    "followSymlinks": true
  }
}
```

### Installed Packages

```coffee
# User
asteroids, v0.2.0
autocomplete-plus, v2.12.0
color-picker, v1.7.0
execute-as-ruby, v0.1.1
file-icons, v1.5.4
filetype-color, v0.1.4
hex, v0.6.1
highlight-line, v0.10.2
language-rust, v0.4.3
linter, v0.12.1
minimap, v4.7.6
vim-mode, v0.44.0

# Dev
No dev packages
```

**Comment by torsten** [comment]
@DomT4 Hi, I released v0.2.1 of the package, it should be fixed now, correct?

**Comment by DomT4** [comment]
Looks like it's running great now, Thanks! :bow:
